main.py

```
@@ -126,64 +127,79 @@ async def prepare_documents(documents: Dict[str, Document]):
-                        }
+                        },
                     )
-                    logger.info(f"Summary length: {len(summary.page_content)}")
-                    for_embed += [summary]
-            except Exception as e:
-                logger.error(e)
-                logger.error(f"Failed to summarize {url}")
+                    logger.info(f"Summary length: {len(summary_text)} chars")
+                    for_embed.append(summary_doc)
+                    summaries.append(summary_text)
+                except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)